### ISSUE TITLE
Gradient visualization

### ISSUE THREAD

**Issue by tonio73** [tonio73/dnnviewer#3]
As an alternative to displaying weights, display gradients : 
- as links between neuron units
- as minimax on the layer details
- as histogram on the unit details
- as map on the convolutional unit details

Graphical design:
- choose distinct colormap (Parula)
- layer minimax, unit histo & maps within a new tab of corresponding subpanel
- links on the main view, plus selector on the right side, see #24

**Comment by tonio73** [comment]
Depends on #22

**Comment by tonio73** [comment]
Hints to get gradients : https://github.com/keras-team/keras/issues/2226

**Comment by tonio73** [comment]
Evaluation of gradients of an existing model:

`(x_train, yTrain), (x_test, y_test) = keras.datasets.cifar10.load_data()`
`with tf.GradientTape() as tape:`
`    y_est = model0(x_train[:1000].astype(np.float)) `
`    objective = model0.loss_functions[0](y_train[:1000], y_est)`
`grads = tape.gradient(objective, model0.weights)`

**Comment by MDESSE** [comment]
- implementation of gradient visualisation based on weights methods.
ISSUE : indexing offset for dense layers
